source/HailPanel.cpp

```diff
@@ -350,6 +350,9 @@ void HailPanel::SetBribe(double scale)
 	for(const shared_ptr<Ship> &it : player.Ships())
 		value += it->Cost();
 
+	if(value < 0)
```

Comment:
Should this be `<=`?

Reply:
Doesnt matter because if its 0 the code after will make it 1000 (and I think its there for that 0 case? Mb put <= and remove it?)

Reply:
```suggestion
	if(value <= 0)
```